catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/workspace/QueryMetacardImpl.java

```diff
@@ -35,6 +52,18 @@ public void setEnterprise(Boolean b) {
         setAttribute(QueryMetacardTypeImpl.QUERY_ENTERPRISE, b);
     }
 
+    public Optional<List<String>> getSources() {
```

Comment:
I would say no need for an optional around the list. The default case of an empty list is good enough.